ocs_ci/templates/app-pods/rhel.yaml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Please name the file also with 7.7 like rhel-7.7.yaml

Reply:
Done